src/components/molecules/MultiStepForm/MultiStepForm.tsx

```diff
@@ -1,25 +1,67 @@
-import React, { useState } from 'react';
-import Button from '../../atoms/Button';
-import { StyledAction, StyledContent, StyledMultiStepForm } from './MultiStepForm.styled';
+import React from 'react';
+import { useSelector } from 'react-redux';
 
-import { IMultiStepFrom } from './MutliStepFrom.model';
+import { useFormProgress } from 'hooks';
+import { RootState } from 'redux/store';
+import { MultistepformNext, MultistepformSubmit } from 'assets';
+import Button from 'components/atoms/Button';
 import Steps from './Steps/Steps';
+import { IMultiStepFrom } from './MultiStepFrom.model';
+import { StyledAction, StyledContent, StyledMultiStepForm } from './MultiStepForm.styled';
+
+const MultiStepForm = ({ onSubmit, children: steps, store }: IMultiStepFrom): JSX.Element => {
```

Comment:
A bit too complicated validation... + redux I recommend this way --> https://codesandbox.io/s/react-hook-form-wizard-form-13yud?file=/src/Step1.js and validation scheme.

Reply:
I exactly used this code before and there was some issues to use it inside our project